### PR TITLE
Added: Basic Home page and routes

### PR DESCRIPTION
**Pull request by KaustubhM1997**
Created a basic Home page UI and added routes

Live preview - https://incandescent-profiterole-11b9b8.netlify.app/

### CODE REVIEW

File: src/pages/Home/home.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      <div className="child-wrapper">
+        <aside className="aside-wrapper">
+          <ul className="aside-list">
+            <li className="aside-item aside-item-active">
+              {" "}
+              <i class="fa-solid fa-house aside-icons"></i>{" "}
+              <a className="aside-link active" href="/">
+                Home
+              </a>
+            </li>
```
Comment (himadri2110) [inline src/pages/Home/home.jsx]: Suggestion: You can move the sidebar into a separate component as it is going to be used on all the pages

**Review by himadri2110 — APPROVED** [review]
Looks good to go.

File: src/pages/Home/home.jsx (hunk 1 above)
Comment (KaustubhM1997) [reply src/pages/Home/home.jsx]: Yes, just pushed the change!

File: src/Routes/routes.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import Mockman from "mockman-js";
+import {Routes, Route} from "react-router-dom";
+import { Home } from "../pages/Home/home";
+import { NoMatch } from "../pages/NoMatch/nomatch";
+import { Login } from "../pages/Login/login";
+import { Signup } from "../pages/Signup/signup";
+import { Archive } from "../pages/Archive/archive";
+import { Label } from "../pages/Label/label";
+import { Trash } from "../pages/Trash/trash";
```
Comment (msdeshmukh009) [inline src/Routes/routes.jsx]: Barrel export here would be very helpful.

File: src/components/Aside/Aside.jsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+          <li className="aside-item">
+            {" "}
+            <i class="fa-solid fa-box-archive aside-icons"></i>
+            <a className="aside-link" href="/archive">
+              Archives
+            </a>
+          </li>
+          <li className="aside-item active">
+            {" "}
+            <i class="fa-solid fa-trash aside-icons"></i>
```
Comment (msdeshmukh009) [inline src/components/Aside/Aside.jsx]: You can replace all the `class` with `className`

File: src/pages/Home/home.css [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+.aside-wrapper {
+  height: 100vh;
+  display: flex;
+  flex: 0.3 1;
+  position: sticky;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
```
Comment (msdeshmukh009) [inline src/pages/Home/home.css]: Ypu can also use `inset: 0 0 0 0;` here

**Review by msdeshmukh009 — APPROVED** [review]
The rest is looking good.

**Pull request merged**